src/components/NavBar/index.js

```diff
@@ -103,18 +104,31 @@ function NavBar() {
 
           <div className={classes.sectionDesktop}>
             <Hidden mdDown>
-              <Button className={classes.signInButton}>
-                <Typography noWrap>Sign In</Typography>
-              </Button>
-              <Button className={classes.signUpButton}>
-                <Typography noWrap>Sign Up</Typography>
-              </Button>
+              {((user === "Undefined") && (user === "null")) ? (
+                <div>
+                  <Button className={classes.signInButton}>
+                    <Typography noWrap>Sign In</Typography>
+                  </Button>
+                  <Button className={classes.signUpButton}>
+                    <Typography noWrap>Sign Up</Typography>
+                  </Button>
+                </div>) : (
+                <Typography className={classes.title} variant="h6" noWrap>
+                  Hi, {user.displayName}
+                </Typography>
+              )}
             </Hidden>
           </div>
-          <Avatar
-            className={classes.avatar}
-            src="https://media-exp1.licdn.com/dms/image/C560BAQHSXxzxdL45FA/company-logo_200_200/0/1588676153700?e=2159024400&v=beta&t=PEiMBARAmeCUpRrAuJZWev-F_oCZac_OunmUXcLuY5U"
-          />
+          {((user === "Undefined") && (user === "null")) ? (
```

Comment:
undefined is a js property, don't put it in quotes.
remove quotes from unefined and null.

Reply:
@Abhishek-kumar09  Sir, i removed the quotes and created new PR in #355.
I request you to check it once.